porcupine/settings.py

```diff
@@ -130,7 +130,7 @@ def add_option(
         calling :func:`set_` with a value that isn't a string or
         calling :func:`get` with the type set to something else than ``str``
         is an error. You can also provide a custom type with the *type*
-        argument, e.g. ``add_option('foo', None, Optional[pathlib.Path])``.
+        argument, e.g. ``add_option('foo', None, Optional[Path])``.
```

Comment:
For docs, I think `pathlib.Path` is better because then it is clear what it means

```suggestion
        argument, e.g. ``add_option('foo', None, Optional[pathlib.Path])``.
```